src/Search.js

```diff
@@ -19,16 +28,85 @@ class Search extends Component {
+                        <img src={searchLogo} alt="Logo" id ="searchLogo"/>
+                        <form onSubmit={this.handleSubmit}>
+                            <div class="form-group">
+                                <label for="inputdefault"></label>
+                                <input class="form-control" id="inputdefault" type="text" placeholder="Type a keyword ... (hike, Chinese, etc.)" value={this.state.value} onChange={this.handleChange}/>
+                            </div>
+                        </form>
+                        <Row>
+                            <label for="sel3">Location</label>
+                            <select class="form-control input-sm" id="sel3">
+                                <option>Seattle, WA</option>
+                                <option>Option 1</option>
+                                <option>Option 2</option>
+                            </select>
+                        </Row>                        
+                        <Row xs={6}>
+                            <p>Price</p>
+                            <Col>
+                                <button type="button" id="dollarButton">
+                                    <span>&#36;</span>
+                                </button>
+                            </Col>
+                            <Col>
+                                <button type="button" id="dollarButton">
+                                    <span>&#36;&#36;</span>
+                                </button>
+                            </Col>
+                            <Col>
+                                <button type="button" id="dollarButton">
+                                    <span>&#36;&#36;&#36;</span>
+                                </button>
+                            </Col>
+                            <Col>
+                                <button type="button" id="dollarButton">
+                                    <span>&#36;&#36;&#36;&#36;</span>
+                                </button>
+                            </Col>
+                        </Row>
+                        <Row>
+                            <p>Distance</p>
+                                <Col>
+                                    <button type="button" >5 miles</button>
+                                </Col>
+                                <Col>
+                                    <button type="button" >10 miles</button>
+                                </Col>
+                                <Col>
+                                    <button type="button" >15 miles</button>
+                                </Col>
+                                <Col>              
+                                    <button type="button" >20 miles</button>
+                                </Col>
+                        </Row>
+                        <div class="text-center">
+                            <button type="button" class="btn btn-warning">Screw The Review!</button>
```

Comment:
If you want to be able to style these past just the defaults, you can set id="str-button" and in App.css use button#str-button{styles here}. You can use this same way for all buttons and give the same id to buttons you'd like styled the same.